Sources/DateTime/InternalUTCClock.swift

```diff
@@ -31,33 +31,60 @@ public struct InternalUTCClock {
         }
 
         public init(secondsComponent: Int64, attosecondsComponent: Int64) {
-            self._value = Swift.Duration(secondsComponent: secondsComponent, attosecondsComponent: attosecondsComponent)
+            _value = Swift.Duration(secondsComponent: secondsComponent, attosecondsComponent: attosecondsComponent)
         }
 
         @available(*, deprecated, renamed: "seconds")
         public func secondsSinceEpoch() -> Int64 {
-            return self._value.components.seconds
+            _value.components.seconds
         }
 
         public func seconds() -> Int64 {
-            return self._value.components.seconds
+            _value.components.seconds
         }
-        
+
         public func attoseconds() -> Int64 {
-            return self._value.components.attoseconds
+            _value.components.attoseconds
         }
     }
 
     public init() {}
 }
 
+private extension String {
+    func pad(_ padding: Int = 2) -> String {
+        let toPad = padding - count
+        if toPad < 1 {
+            return self
+        }
+        return "".padding(toLength: toPad, withPad: "0", startingAt: 0) + self
+    }
+}
+
+extension InternalUTCClock.Instant: CustomStringConvertible {
+    public var description: String {
+        var epoch = EpochDateTime.unixEpoch()
+        epoch.convert(timestamp: Int(seconds()))
+
+        return """
+        \(epoch.year)-\
+        \(String(epoch.month).pad())-\
+        \(String(epoch.day).pad()) \
+        \(String(epoch.hour).pad()):\
+        \(String(epoch.minute).pad()):\
+        \(String(epoch.second).pad()).\
+        \(String(attoseconds() / 1_000_000_000_000)) // output with microseconds
```

Comment:
I think this also should be padded to 6 symbols?

Reply:
It will always be 6 digits, as we start with a very large value (attoseconds) and just divide it - it will have trailing zeroes automatically.

Reply:
Fixed, although very unlikely it would be an issue in reality... Added unit test as well.